plugin/src/main/java/io/jenkins/plugins/casc/impl/secrets/PropertiesSecretSource.java

```diff
@@ -36,26 +36,26 @@
 
     @Override
     public Optional<String> reveal(String secret) {
-        // lazy initialization
-        if (secrets == null) {
-            secrets = new Properties();
-            final String secretsEnv = System.getenv("SECRETS_FILE");
-            final String secretsPath = secretsEnv == null ? SECRETS_DEFAULT_PATH : secretsEnv;
-            final File secretsFile = new File(secretsPath);
-            if (secretsFile.exists() && secretsFile.isFile()) {
-                try (InputStream input = new FileInputStream(secretsFile)) {
-                    secrets.load(input);
-                } catch (IOException ioe) {
-                    LOGGER.log(Level.WARNING,
-                        "Source properties file " + secretsPath + " could not be loaded", ioe);
-                }
-            }
-        }
-
         if (secrets.getProperty(secret) == null) {
             return Optional.empty();
         } else {
             return Optional.of(secrets.getProperty(secret));
         }
     }
+
+    @Override
+    public void init() {
+        secrets = new Properties();
```

Comment:
Perhaps this should be initialized on the field and than use `clear` 
```suggestion
        secrets.clear();
```

Reply:
Agree, thank you, fixed